lib/screens/components/meansStatistics.dart

```diff
@@ -78,6 +78,12 @@ class MeansStatistics extends StatelessWidget {
                 child: charts.TimeSeriesChart(
                   _createSampleData(),
                   animate: true,
+                  domainAxis: new charts.DateTimeAxisSpec(
```

Comment:
Please remove 'new'. 
```suggestion
                  domainAxis: charts.DateTimeAxisSpec(
```